qdrant-landing/content/blog/qdrant-1.8.x.md

```diff
@@ -0,0 +1,130 @@
+  - vector search
+  - new features
+  - sparse vector performance
+  - dymamic CPU saturation
+  - text immutability
+  - text loading optimization
+weight: 0 # Change this weight to change order of posts
+---
+
+Today, we are pleased to announce the release of [Qdrant 1.8.0](https://github.com/qdrant/qdrant/releases/tag/v1.8.0).
+We've optimized performance in a number of ways. We've addressed issues with:
+
+<!-- Shard deltas (deferred to v1.9) -->
+- Sparse vectors
+- Dynamic CPU saturation
+- Text immutability
+- Text loading optimization
+
+We've also added <!-- TBD -->.
+
+Did we miss something? We rely on your feedback to drive our development. We
+welcome your contributions, especially in our [Discord community](https://qdrant.to/discord). Join us, introduce yourself, and help us build the best vector search engine!
+
+## Improved sparse vector performance
+
+The primary focus of Qdrant is performance. It's why we build our work in Rust.
+We also want to optimze performance in "real-world" situations.
+
+We used the [NeurIPS 2023 datasets](https://big-ann-benchmarks.com/neurips23.html). We've optimized our search to improve throughput by a factor of 16.
+
+We set up test benchmarks in our [Sparse vectors benchmark](https://github.com/qdrant/sparse-vectors-benchmark) repository. For our benchmarks, we use
+moderately-sized Azure instances with separate clients and servers:
+<!-- Arnaud isn't sure we should include these details. I think they're useful for our readers. (I've removed the Azure configuration bits described elsewhere.)  -->
+
+| System | vCPU | RAM (GB) |
+|--------|------|----------|
+| Server | 8    | 32       |
+| Client | 4    | 16       |
+
+We created the test collection with:
+
+- 8 Sparse NeruIPS segments
+- `index.on_disk=false`
+
+Based on those datasets, we found the following two-dimentional histogram of
+latency per query dimension count:
+
+![Histogram with increasing latency for higher query dimensions](/blog/qdrant-1.8.x/neurIPS_bench_example.png)
+
+The colors within the scatter plot show the frequency of the results. The "red"
+points show the highest frequency.
+
+While this is a two-dimensional graph, the dataset represents over 100 dimensions! 
+
+## Optimized CPU use
+
+We continue to optimize our search to minimize the load on your hardware. One
+part of that is on CPUs. On a typical system, the loads on each CPU is 
+low, which is a waste of resources.
+
+We optimize how we use CPUs. With dynamic CPU saturation, we set an 
+`optimizer_cpu_budget` to drive the number of CPU _cores_ to saturate with
+optimization tasks. Specifically, if the value is:
+
+- `0`: Qdrant keeps one or more CPU cores unallocated (default)
+- A negative number: Qdrant subtracts this from the number of available CPU cores
+- A positive number: Qdrant assigns this exact number of CPU cores to your configuration
+
+<!-- Question: where do we set the `optimizer_cpu_budget, and how does that relate to `max_indexing_threads` -->
```

Comment:
In the configuration (`config.yaml`). There it also explains what the possible values are. It is very likely that users keep it at 0, which means: automatic.

See: https://github.com/qdrant/qdrant/blob/61fa4e74e4de9f08393a77a14489983a32700c85/config/config.yaml#L48-L52

Around it, there are other parameters that affect CPU utilization as well. But those have existed before.

Reply:
Based on your other comments, I'm guessing you're just answering my question. While I still reference the `optimizer_cpu_budget`, I've removed details. Users can search for that parameter in our code.

Reply:
> Users can search for that parameter in our code.

In our configuration file, yes!